src/plugins/dashboard/public/dashboard_app/_dashboard_app_strings.ts

```diff
@@ -154,6 +154,11 @@ export const shareModalStrings = {
       defaultMessage:
         'One or more panels on this dashboard have changed. Before you generate a snapshot, save the dashboard.',
     }),
+  getDraftShareWarning: () =>
+    i18n.translate('dashboard.snapshotShare.draftWarning', {
+      defaultMessage:
+        'One or more panels on this dashboard have changed, save the dashboard to create a permanent link.',
+    }),
```

Comment:
Should this warning also show up in embed mode?

![Oct-24-2024 10-04-07](https://github.com/user-attachments/assets/ef8df143-2d85-4732-8351-a3e2c6a5d553)

Reply:
Bumping this 🙇

Reply:
Hey... saw this, I've added in a message for this the embed tab too.